src/AMD.jl

```diff
@@ -144,7 +145,7 @@ for (orderfn, typ) in ((:_amd_order, Cint), (:_amd_l_order, _Clong))
       valid = ccall($orderfn, $typ,
                     ($typ, Ref{$typ}, Ref{$typ}, Ptr{$typ}, Ptr{Cdouble}, Ptr{Cdouble}),
                      nrow, colptr,    rowval,    p,         meta.control, meta.info)
-      (valid == AMD_OK || valid == AMD_OK_BUT_JUMBLED) || throw("amd_order returns: $(statuses[valid])")
+      (valid == AMD_OK || valid == AMD_OK_BUT_JUMBLED) || throw("amd returns: $(amd_statuses[valid])")
```

Comment:
Why change `amd_order` here?

Reply:
Because `colamd` and `symamd` are also `amd_order`.

Reply:
Do you prefer `amd_order`?

Reply:
It's the name of the function we call here.

Reply:
Ok, I change that.